docs/settings.md

```diff
@@ -28,12 +28,21 @@ For example, in case you want to run the app on port `5000`, just set the enviro
 
 ## Development
 
-* `--reload` - Enable auto-reload.
+* `--reload` - Enable auto-reload. Uvicorn supports two versions of auto-reloading behavior enabled by this option. There are important differences between them.
 * `--reload-dir <path>` - Specify which directories to watch for python file changes. May be used multiple times. If unused, then by default the whole current directory will be watched. If you are running programmatically use `reload_dirs=[]` and pass a list of strings.
+
+### Naive Reloading
+
+If Uvicorn _cannot_ load [watchgod](https://pypi.org/project/watchgod/) at runtime, it will periodically look for changes in modification times to all `*.py` files (and only `*.py` files) inside of its monitored directories. See the `--reload-dir` option. Specifying other file extensions is not supported unless watchgod is installed. See the `--reload-include` and `--reload-exclude` options for details.
+
+### watchgod-Enabled Reloading
+
+For more nuanced control over which file modifications trigger reloads, install [watchgod](https://pypi.org/project/watchgod/) where Unvicorn can see it. Alternatively, install `uvicorn[standard]`, which includes watchgod as a dependency. This will enable the following options (which are otherwise ignored).
```

Comment:
I'd suggest the opposite, 1st install `uvicorn[standard]` and then watchgod manually